de-lib/src/main/java/org/iplantc/de/pipelines/client/views/PipelineAppOrderViewImpl.java

```diff
@@ -115,6 +141,21 @@ public PipelineTask getOrderGridSelectedApp() {
         return appOrderGrid.getSelectionModel().getSelectedItem();
     }
 
+    private void setButtonState() {
+        int size = appOrderGrid.getSelectionModel().getSelectedItems().size();
+        if(size > 0) {
+            removeAppBtn.enable();
+            if(appOrderGrid.getStore().size() > 1) {
+                moveDownBtn.enable();
+                moveUpBtn.enable();
+            }
+        } else {
+            removeAppBtn.disable();
+            moveUpBtn.disable();
+            moveDownBtn.disable();
+        }
+    }
```

Comment:
Ah, I missed that `size` referred to the size of the selection model. I assume that this is the number of items currently selected in the list (which is at most one because this is a single-select list).